cmake/projects/wavm/hunter.cmake

```diff
@@ -15,9 +15,9 @@ hunter_add_version(
     VERSION
     1.0.2
     URL
-    "https://github.com/soramitsu/WAVM/archive/7efbcced0d41d5f7bc6cd254d624e5f7174b54fc.tar.gz"
+    "https://github.com/soramitsu/WAVM/archive/refs/tags/1.0.3.zip"
```

Comment:
Hm... Tag 1.0.3, but version 1.0.2
Is it correct?

Reply:
Nope.